src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -225,7 +227,7 @@
   private   ImageButton                 attachButton;
   protected ConversationTitleView       titleView;
   private   TextView                    charactersLeft;
-  private   ConversationFragment        fragment;
+  public   ConversationFragment        fragment;
```

Comment:
Variable name not aligned with the rest

Reply:
Why was this changed from private to public? Attributes are usually never public.

Reply:
@erinb90  I agree, I will investigate